src/routes/index.ts

```diff
@@ -1,10 +1,12 @@
-import { Router } from "express";
-import  locationRoutes  from "./location";
-import eventRoutes from "./event";
+import { Response, Request } from "express";
+import prompt from "../Utils/apiResponses";
 
-const router = Router();
+const baseUrl = "/api/v1/";
 
-router.use('/locations', locationRoutes);
-router.use('/events', eventRoutes);
-
-export default router;
\ No newline at end of file
+export class TestRoute {
```

Comment:
my favorite question, why is this a class?
```suggestion
const router = Router(); // this is a global singleton, means you don't have to keep passing `app` to methods
router.use('/locations', locationRoutes); //etc...no repeating base url everywhere
```

Reply:
> Great job @ywalakamar, the refactor looks amazing! I have raised a few concerns mainly with the routing, which I hope we can centralize. If you can look into that, we can merge this in!

Great reviews @fatmali, thanks, WIP!